### PR TITLE
CRM: Add new Support Contact page

### PR DESCRIPTION
**Pull request by cleacos**
## Proposed changes:

This PR adds the new Support Contact page. If the user has a valid license, they can send a support ticket directly to Zendesk.

**(1) Without license:**

image

**(2) With an invalid license:**

image

**(3)With a valid license:**

image

If the user has a license, this `Support` menu is added:

image

Also, the top JPCRM menu was updated for any user:

image

**Actions available**
- Click on 'Support` WP menu or JPCRM top menu will send you to the Support page `wp-admin/admin.php?page=jpcrm-support`
- Click on `Knowledge Base` will send you to https://kb.jetpackcrm.com page (new tab)
- Click on `Support Forum` will send you to https://wordpress.org/support/plugin/zero-bs-crm (new tab)
- With a valid license, it'll show you the support contact form. Filling it out, and clicking on `Submit` will create a ticket on Zendesk with a summary of technical data from the site.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

( [Go to the issue](https://github.com/Automattic/zero-bs-crm/issues/2983) to see the instructions there )

### CODE REVIEW

**Comment by coder-karen** [comment]
For some reason when testing locally and clicking submit to send a ticket I get a notice saying 'Error! Invalid license' despite the license being valid (and saying that it is valid at the top of the support page).
I then tried the same on my Pressable site with this branch checked out and got  the same error (it also has a valid licence). In both cases the URLs are changed in ZeroBSCRM.Core.php.

**Comment by cleacos** [comment]
@coder-karen did you generate the license in our dev server?

File: projects/plugins/crm/admin/support/main.page.php [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+			<span class="expires">
+				<?php echo esc_html__( 'Expires', 'zero-bs-crm' ); ?>:
+				<?php echo ( isset( $license['expires'] ) ? esc_html( gmdate( 'Y-m-d H:i', $license['expires'] ) ) : '-' ); ?>
+			</span>
+			<span class="account">
+				<?php // translators: %s: URL to account page. ?>
+				<?php echo wp_kses_post( sprintf( __( '<a href="%s" target="_blank">Go to your Jetpack CRM account</a>', 'zero-bs-crm' ), esc_url( $zbs->urls['account'] ) ) ); ?>
+			</span>
+		<?php else : ?>
+			<span class="label label-danger"><?php echo esc_html__( 'NO VALID', 'zero-bs-crm' ); ?></span>
```
Comment (coder-karen) [inline projects/plugins/crm/admin/support/main.page.php]: Minor thing here:

```suggestion
			<span class="label label-danger"><?php echo esc_html__( 'NOT VALID', 'zero-bs-crm' ); ?></span> 
```

**Review by coder-karen — DISMISSED** [review]
Works well, very nice!

Left a minor suggestion but still approving.

**Review by coder-karen — DISMISSED** [review]

**Comment by tbradsha** [comment]
I haven't reviewed the code and won't jump in there today, as @coder-karen has, but noting that it should be "Not valid" instead of "No valid". :^)

**Comment by cleacos** [comment]
That was Karen's suggestion. Commit here: https://github.com/Automattic/jetpack/pull/29545/commits/b78784659017911c459758b86a6b9da0510991a9

**Comment by cleacos** [comment]
I have to merge/rebase this PR. And resolve the conflict.

**Review by coder-karen — APPROVED** [review]

**Pull request merged**